web/styles/dark_theme.css

```diff
@@ -779,7 +779,7 @@
 
     #topics_header {
         .show-all-streams {
-            color: hsl(236deg 33% 90%);
+            color: hsl(0deg 0% 100% / 75%);
             opacity: 0.75;
```

Comment:
Can we strike this separate `opacity` property, since presumably that's now being expressed with the `hsl()` function?